src/webapp/components/collapsible-section/Section.tsx

```diff
@@ -13,53 +13,62 @@ export interface SectionProps {
     elevation?: number;
     iconPos?: iconPosition;
     classProps?: {
-        section?: string;
-        header?: string;
-        content?: string;
+        sectionPaper?: string;
+        sectionHeader?: string;
+        sectionContent?: string;
     };
     arrowStyle?: arrowStyle;
 }
 
-export const Section = ({
-    children,
-    title,
-    collapsible,
-    isOpen = true,
-    setOpen = () => {},
-    elevation = 1,
-    iconPos = "right",
-    classProps = {},
-    arrowStyle = "upDown",
-}: SectionProps) => {
+export const Section = (props: SectionProps) => {
+    const {
+        children,
+        title,
+        collapsible,
+        isOpen = true,
+        setOpen = () => {},
+        elevation = 1,
+        iconPos = "right",
+        classProps = {},
+        arrowStyle = "up_down",
+    } = props;
     const classes = useStyles();
     const leftIcon = iconPos === "left" ? classes.leftIcon : null;
     const toggle = () => setOpen(!isOpen);
 
+    const paperClasses = useMemo(() => ( _.compact([classes.paper, classProps.sectionPaper]).join(" ")),
+        [classes.paper, classProps.sectionPaper]);
+    const headerClasses = useMemo(() => ( _.compact([classes.header, leftIcon, classProps.sectionHeader, (isOpen ? null : classes.noBorder)]).join(" ")),
+        [classes.header, leftIcon, classProps.sectionHeader, isOpen, classes.noBorder]);
+    const contentClasses = useMemo(() => (_.compact([classProps.sectionContent, (isOpen ? null : classes.hidden)]).join(" ")),
+        [classProps.sectionContent, isOpen, classes.hidden]);
+
```

Comment:
No need for `useMemo` here, as the final value is a string (and the computation is very light)

I'd create a simple function (in src/webapp/utils/react.ts?) so you can write `className={getClassName([classes.paper, classProps.sectionPaper]}`